### DISCUSSION TITLE
"The YouTube API key you provided was not a string."

### DISCUSSION THREAD

**Discussion by yeehaw2** [eritislami/evobot#871] [Q&A]
Hello, I have a bit of a problem trying to get the bot to work on discord. Whenever I use node index.js it tells me "The YouTube API key you provided was not a string.". Heres a picture of how my config looks like.
![image](https://user-images.githubusercontent.com/36142215/133130645-d8186d3f-6344-47ba-af57-5e171b04a2e9.png)

**Comment by eritislami** [comment]
Missing a quote on "SOUNDCLOUD_CLIENT_ID:

**Reply by yeehaw2** [reply]
Ah, thank you, and also, is there anyway I can fix this? whenever I use "/help" it doesn't respond and leads to this 
![image](https://user-images.githubusercontent.com/36142215/133139654-20813d2a-945d-48d7-8e53-95c52e9c88d8.png)

**Reply by eritislami** [reply]
Give the bot admin permissions, or specific permission for all the features, view channels, send a message, join voice etc...

**Reply by KUFTATV** [reply]
can you show me an example since this is my config
![Capture d’écran 2021-09-14 161015](https://user-images.githubusercontent.com/90708413/133284118-fb4f4318-fbc1-493b-b417-423ad2889c9e.png)

**Reply by eritislami** [reply]
Delete the node before the opening bracket node{ it should start with just {

**Reply by eritislami** [reply]
> can you show me an example since this is my config

Please take time to actually read the README, it will solve 99% of problems...

**Comment by KUFTATV** [comment]
thx i solved it

Le mar. 14 sept. 2021 à 17:52, Erit Islami ***@***.***> a
écrit :
